Bottom: 34b9933cf18a91e480cb9f9a2264e6b218c9dfb4
Top:    6f6b5d4aacc3b1e8114e5f441860e70a333e9637
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-23 15:19:09 -0500

Refresh of check-first-install

---
diff --git a/roles/nexus3/tasks/nexus_install.yml b/roles/nexus3/tasks/nexus_install.yml
index 22c68332..02053005 100644
--- a/roles/nexus3/tasks/nexus_install.yml
+++ b/roles/nexus3/tasks/nexus_install.yml
@@ -217,7 +217,7 @@
   file:
     path: "{{ nexus_data_dir }}/clean_cache"
     state: touch
-  when: nexus_latest_version.changed and nexus_data_dir_contents.stdout != ""
+  when: nexus_latest_version.changed and not nexus_first_install
   tags:
     # hard to run as a handler for time being
     - skip_ansible_lint
@@ -260,12 +260,12 @@
 - name: First-time install admin password
   set_fact:
     current_nexus_admin_password: 'admin123'
-  when: nexus_data_dir_contents.stdout == ""
+  when: nexus_first_install
 
 - name: Subsequent re-provision admin password
   set_fact:
     current_nexus_admin_password: "{{ nexus_admin_password }}"
-  when: nexus_data_dir_contents.stdout != ""
+  when: not nexus_first_install
   no_log: true
 
 - name: Create directory to hold current groovy scripts for reference
